.travis.yml

```diff
@@ -49,8 +49,7 @@ install:
 
 script:
 - set -eo pipefail
-- make protolint;
-- make protodocs;
+- make protoc;
```

Comment:
`protoc` also creates the Go files from protobuf. This is an antipattern as the following build steps do not run on the commited code anymore. `protolint` should be enough to enforce linter policies. `protodocs` is wrong as well as the  docs are part of the repo (right now).

Reply:
Thanks. I will modify `protodocs` command to generate documentation and then publish it to docs.iov.one